config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml

```diff
@@ -216,6 +216,8 @@ periodics:
             value: (\[sig-windows\]|\[sig-scheduling\].SchedulerPreemption|\[sig-autoscaling\].\[Feature:HPA\]|\[sig-apps\].CronJob).*(\[Serial\]|\[Slow\])|(\[Serial\]|\[Slow\]).*(\[Conformance\]|\[NodeConformance\])|\[sig-api-machinery\].Garbage.collector
           - name: GINKGO_SKIP
             value: \[LinuxOnly\]|device.plugin.for.Windows|RebootHost
+          - name: GINKGO_NODES
```

Comment:
Should we add this to `preset-capz-serial-slow`?
We set CONFORMANCE_NODES=1 there.

Reply:
Maybe we should add another preset then?
I'm worried that future serial jobs will just copy the presets and miss this env var

Reply:
discussed offline and decided to combine